src/spotbugs/excludesFilter.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+
+  <Match>
```

Comment:
would be nice to get some short explanations for each exclusions, but I'm not insisting, especially if it's temporary

can you comment at least on the cases where there are open bugs in spotbugs or we don't intend to fix the violations in near future?

Reply:
I've added short explanations. I'll post a comment on the status of every exclusion.